src/clients/api/queries/getMarketHistory/index.ts

```diff
@@ -0,0 +1,78 @@
+import { VError } from 'errors';
+import { MarketSnapshot, VToken } from 'types';
+import { restService } from 'utilities';
+
+const ENTRIES_HOUR = 2;
+export const ENTRIES_DAY = ENTRIES_HOUR * 24;
+const ENTRIES_WEEK = ENTRIES_DAY * 7;
+const ENTRIES_30_DAYS = ENTRIES_DAY * 30;
+const ENTRIES_YEAR = ENTRIES_DAY * 365;
```

Comment:
A small thing, but I'd add `PER_` to each constant name to make it extra clear what they represent. I'd also export all of them, incase they might be used from somewhere else.

```suggestion
export const ENTRIES_PER_HOUR = 2;
export const ENTRIES_PER_DAY = ENTRIES_PER_HOUR * 24;
export const ENTRIES_PER_WEEK = ENTRIES_PER_DAY * 7;
export const ENTRIES_PER_30_DAYS = ENTRIES_PER_DAY * 30;
export const ENTRIES_PER_YEAR = ENTRIES_PER_DAY * 365;
```

What do you think?

Reply:
Taken this, it's clearer. I also removed the export for `ENTRIES_PER_DAY`, my first idea was to use it in the chart, but it ended up feeling a bit strange to have this value coming out of the query.

Reply:
Makes sense